src/libraries/System.Composition/src/System.Composition.csproj

```diff
@@ -8,7 +8,7 @@
     <NoWarn>$(NoWarn);NU5128</NoWarn>
     <!-- Reference dependencies to persiste them in the project's project.assets.json file which is used to calculate package dependencies. -->
     <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
-    <PackageDescription>Provides a version of the Managed Extensibility Framework (MEF) that is lightweight and specifically optimized for high throughput scenarios, such as the web.</PackageDescription>
+    <PackageDescription>Provides the foundational attributes that allow you to declare parts for composition, such as imports, exports, and metadata with the Managed Extensibility Framework (MEF).</PackageDescription>
```

Comment:
Isn't it supposed to be added/updated for System.Composition.AttributedModel.csproj?

https://github.com/dotnet/runtime/blob/f2d5480e5f1e0ade3e73cdd176151f625d6db278/src/libraries/System.Composition.AttributedModel/src/System.Composition.AttributedModel.csproj#L8-L18

Reply:
You're right, something must have gone wrong. Will update this.